addons/msg-title/favicon.js

```diff
@@ -0,0 +1,21 @@
Ij24k/8zXu9kyJvV2g2lyrES1YP038JUUSLh0kaW+3xDuWSDANcQ2FEJrDBR9njQj9anGZlxFpAyaB+tsU12yQcPkw8cY8yrxnyjgn9id1YhRmSFS7VaLq9Uy9UgnJKVEUMBNryBUbyehrTb6Ity1AyJ/3E/IIQj6pF0KfOhpa7vzL0Wm60chVQOuo+tQb71wmwfz/WF8outIKYVr5X74u8Z5tiKgin5Qaf/C/+AfTrETi0LdlCqJM4txS4YTcvE8ICTJc2C9LCAGmliVB852dtq4Fm0zLzszdnP2WeCOIOuob6uxMsWDO2xLr3d80W4KZ3XD8G6Su3/niM+XiLQsgBjn+b4XMlWBql2whYx6TcPFnDYXEGikZldcdnaYb7V0PdpmWnRkS3H1v2K2QcCFXhcztSefUVVpTTX+F6UZo/exCM/18kmW0pmyfkM2I1ezfUEjZ4xJWDkBIffhFQ7yN026OTtONxnSlZ1t2ZkjoHR/qSKgQwmhQ0YvOmf+bZkkwo6sNxZpbmH51A9qY0qV+am2e01gISTUkIDQQoqF3I3jF0Wm6MT+1aKszLTuLN5cj5O+mrKgXMuw2hHxohWz8khH5qxViRkyDQVtG8TWJdq5ATEjUm2t8KEvIuCeJaoNFwiwhazl92dFpsh1BQ/fnJGT47RLM6+2EzLSOrBneCNHwrEQ7SrB8qMR+ramQrRDXhv4+/ikTDSW0yxv61dC7FHR2fJpujMhdNJS1cEqcWjoi8z6wQozQvyEEH9HaSwOCXjfidvHxnYgRC1ePNsJMCZPJ7NO7Srhhuqka1PIqhEZuBPtSLRZvYX4zTbKcfehNOPs75Aicfedyzt/ld5cjVIwWjqbWuod8Q+ky4SlG8TaTf2zku0r8hR9LVLcTBXZdQr8ZH2l60Ug7Oq2+w/EZjtkjwrTQUGvDL+WJ+sCQGyT4qrvJ2mmyerR1HvN9IHmhj8k5/tz3SKCPE8WoiLXu+hz/yVS/pmhECP+Pti+ig9D1pGJMrfUhp01b8vJHFXIr6A9sXe0sYmppp1ZIpli8ljnfmWkzi6p4pxkZkwRZk8f+Hgn2psRnJag5xfiF1mX6P+M/JFXjQ1S/S4tY+mYFSR0Z3dzQraYTHb3cjf8fyR+fI46Xc7RLWu6UlicBSUtXdbY8d0J0euD8Om0C8kW4lwzPslfFGEFkQyNq9zqyfoktGAfr/6+U2k8ulWWPtpapF54mE887U0pPOklKTz5ZxrZuLbPatZNFXbqkUrNn96ONHzl6uZsT8irQvSY7abVYpAQPZv3TRiz1DSNEnR1/GXm31BXfJx4Z28dfggW9QR8JlxGpdmuNRQ5BkFa/4dpxUlvaSebdca6UHHeUjKDLQ6HsnHMKh7do0dpRzM2ckDeB7lVpDjPxPaqaYXPEQJ3fuphiJLRgnPIStdYb4hFCzRojszIkEWpVHCz+lIJyCvljjqQJv1vLiqX0xOMTCR8KiGnnaB7anJCeYEe9EGqgCDImIzOnDXQ5i+NG68aZPatw2WCKv4fF63+RFamOrE498k7zvU9+WdHtWTN9kkjmCsT83lE9uDkh74BUtpDJErBUzTio1/9i4yvxjkpTekS68tP80J8RyWws6PRz6/hVz14kpSe0SiT3TZHTyMDpaPj34rhvnW6ETLJCClUIUIIqZGuFmf/RBhZQE57hdxcEGiD17sVS8u2jE0k1EalD+gy0jyNifQA53cvdJ+RrFfInS1SF6J2nXI8oAvWimMWTLld1A85c48oPvbb8utOSyDQXfR3lZIPTKQj5CNRkC5kogS5xTRJDiGZ2yhNdKKmZzM4yOKj4t/jF91r/QEjqvTb5Ho16HHRU4K677X2Bbsc0FlKgGw6a1cnSpQ+R7cnImF5qR4Z8M+998cramyi2rMMFiSTyAYR0crSzDR6a1c/mqM8q9AFNghAXkTQpUn7HO5aaS3SFaMRoMkytNgVitG68fHnzNYkk8gS0JBh8NGKdAz4DBxECdBdlwtOQXm4uiWs3SbRtIVhEqbLLLqSYbpqlEwjkCylHPdvgo0LOBjoiB5ha+41IaqUOB1GLhFnOep4y3ienBGvGmB35pia/XOGoNza4H8RHOtnRMD5CeVL6gMS68lMhEPcZobrhlPDjnhB/4jOABHj8sYkE8gVHu7HB/VTwEdgvamXCLyI0aqkg1h0h30e71khY+an4ox60iVLX6xrVQNkZLRMJ5AlrHO3GBvdjgT7cbJDZqbVIeMFUlrFFWjBqfiBXsGAyd37ay+K7KFUvVH8HUy84IYlAvlDhaDc2aB9JDukOquqFsATVbf9gdg9b7WrGNtMLQer0rBLNhoM+H1EBZjR0Cl5BmX52EoG8gJB14O0iFQL/zgiZy9GuR3Q97qUkWDGcKvePlrwSNiWI1lWutjIgv+h6w5X3tUV3JJLIBxByiaPd2KB9BNDn5+rw+54VMjhRaoX4SwpIdo+5xRECdGTqpxOfS34r/vyP8Z0pEhOOdXdyzoPtEok0E8k5JGNQ1qSobyB05LjOiHBmduN1/3dRP/HxF183s0tYUJVQkoyiVB//tH1qu3OV7Hu8iYulUklEmgMtGg88Gg0NERcCfQCatf9rphlrc12/h5WFprYyj50RF66fZjaydUW4v20cPjyJUJOAiOTSJMkQcCK4ExTiL7oPm2X6EkC6psq8AGBeBKheb3ZRNKccyFb06BEmEfuG6Oko5mb0q1NMxTyCEH2+vgEcmGWCcb3mIn0BYCHQN4XGLX311RkJ5HKBTqfcR2J/g4Bm+rsg0RdQj+Rm/Keaw2SOuj+mu5Z38lmDyOUV7dv/urhVq4IEsgfCpJx94kBG5/oijTr/9RyfB5+AaWANwmqAbqQpzDtYYA4YAnQn5l7+dy44hXN9H+skjq20TW1byeldVqL7EU/pd/zWrdkCkgwS+taOPm54CwFTwSbOPaDvX20Fc8F/+awPUc/LED681qLF/wBb6icWdpxK1gAAAABJRU5ErkJggg==`;
```

Comment:
This icon is really blurry

Reply:
it's the same one scratch normally uses, except with a red dot on it

Reply:
Wait, you coded this?

Reply:
basically yeah

on the discord he just asked "what next?", I figured it out, and he added it to his branch